### PR TITLE
Bug: fix getCalendar day cursor in short months

### PR DESCRIPTION
**Pull request by sallaben**
- Create day cursor in local timezone

For more discussion see #243

### CODE REVIEW

**Comment by sallaben** [comment]
Thanks, github-actions bot. Verified the change works on the 244 docs as well. 
image

File: attractions/utils/datetime-utils.js [hunk 1]
```diff
@@ -215,12 +215,12 @@ export function datesLessEqual(date1, date2) {
...
   const calendar = [];
-  const dayCursor = new Date(0);
-  dayCursor.setFullYear(year, month);
+  const dayCursor = new Date();
+  dayCursor.setFullYear(year, month, 1);
 
   // Offset the start of the month to the closest left `firstWeekday`
   dayCursor.setDate(
-    1 - ((daysInWeek + dayCursor.getDay() - firstWeekday) % daysInWeek)
+    0 - ((dayCursor.getDay() + firstWeekday) % daysInWeek)
```
Comment (aabounegm) [inline attractions/utils/datetime-utils.js:223]: I understand the motive behind removing `daysInWeek`, it does seem redundant with the modulo, but it is actually needed due to the way JavaScript calculates the modulo of negative numbers (which remains negative). Also, why did you change the `1` into `0` and `-` into `+`? It now marks today (15th of February) as Thursday instead of Monday for me (I currently live in UTC+3 timezone, btw).

image

**Review by aabounegm — CHANGES_REQUESTED** [review]
It seems that the same issue affects the weekday labels as well. I suggest changing the `new Date(0)` in line 148 of the same file to `new Date(1970, 0, 1)`

File: attractions/utils/datetime-utils.js (hunk 1 above)
Comment (sallaben) [reply attractions/utils/datetime-utils.js]: This change was erroneous -- a blind attempt to match the dates to the weekday labels that displayed on my screen. 

Your comment pointing out that my week started on Friday clued me in that the labels on my screen were also being calculated wrongly. You also pointed out that getWeekdays also used new Date(0). Once I updated getWeekdays to use new Date(1970, 0, 1), the labels matched up and I saw that the change to the logic here wasn't warranted.

File: attractions/utils/datetime-utils.js [hunk 2]
```diff
@@ -215,12 +215,12 @@ export function datesLessEqual(date1, date2) {
  */
 export function getCalendar(month, year, firstWeekday) {
   const calendar = [];
-  const dayCursor = new Date();
-  dayCursor.setFullYear(year, month, 1);
+  const dayCursor = new Date(1970, 0, 1);
+  dayCursor.setFullYear(year, month);
```
Comment (aabounegm) [inline attractions/utils/datetime-utils.js]: Couldn't these 2 lines be combined as such:
```suggestion
  const dayCursor = new Date(year, month, 1);
```
?

File: attractions/utils/datetime-utils.js (hunk 1 above)
Comment (aabounegm) [reply attractions/utils/datetime-utils.js]: Cool! Thanks for **your** help in finding and fixing bugs!

File: attractions/utils/datetime-utils.js (hunk 2 above)
Comment (sallaben) [reply attractions/utils/datetime-utils.js]: Thanks, I did not know about that equivalency! Committed your suggested change.

**Review by aabounegm — APPROVED** [review]
LGTM

**Review by illright — APPROVED** [review]

**Comment by aabounegm** [comment]
Thank you so much for your help, @sallaben!
